api-docs/api-reference/methods/post-create-backup-with-copy.rst

```diff
@@ -0,0 +1,135 @@
+
```

Comment:
Most of this is covered in another rst file for backups. The only addition is the copy-to parameter in the payload.
Should we consider just modifying the existing rst to cover copy to functionality?

Reply:
Nevermind, it looks like separate files were provided for both instance and HA groups even though the only difference is the instance/source parameters in the payload. <sign>